### ISSUE TITLE
OPDS error on "Recently Added" tree

### ISSUE THREAD

**Issue by barbequesauce** [evilhero/mylar#1824]
Using Marvin3, I'm getting an error when attempting to access the "Recently Added" tree. The error reads: "Oops Cannot download feed. The service might not be available or you are not connected to the internet.".

All other trees seem to function without issue.

**Comment by barbequesauce** [comment]
Debug log snippet here. Note that there's no reference to recently added.

```
28-Dec-2017 15:46:54 - DEBUG   :: CP Server Thread-10 : Recieved OPDS command: root
28-Dec-2017 15:46:54 - DEBUG   :: CP Server Thread-10 : {u'58442': u'58442', u'59509': u'59509', u'59354': u'59354'}
28-Dec-2017 15:47:02 - DEBUG   :: CP Server Thread-10 : Recieved OPDS command: Recent
28-Dec-2017 15:47:09 - DEBUG   :: CP Server Thread-10 : Recieved OPDS command: Recent
28-Dec-2017 15:50:40 - DEBUG   :: Thread-13 : Refresh sequence set to fire every 5 minutes for 7 day(s)
28-Dec-2017 15:50:40 - DEBUG   :: Thread-13 : ReScheduled job: DB Updater to 2017-12-28 20:55:40
28-Dec-2017 15:50:40 - INFO    :: Thread-13 : Update complete
28-Dec-2017 15:53:16 - DEBUG   :: CP Server Thread-11 : Recieved OPDS command: Publishers
28-Dec-2017 15:53:18 - DEBUG   :: CP Server Thread-11 : Recieved OPDS command: Publisher
28-Dec-2017 15:53:21 - DEBUG   :: CP Server Thread-11 : Recieved OPDS command: Publishers
28-Dec-2017 15:53:23 - DEBUG   :: CP Server Thread-11 : Recieved OPDS command: root
28-Dec-2017 15:53:23 - DEBUG   :: CP Server Thread-11 : {u'58442': u'58442', u'59509': u'59509', u'59354': u'59354'}
28-Dec-2017 15:53:24 - DEBUG   :: CP Server Thread-11 : Recieved OPDS command: StoryArcs
28-Dec-2017 15:53:27 - DEBUG   :: CP Server Thread-11 : Recieved OPDS command: root
28-Dec-2017 15:53:27 - DEBUG   :: CP Server Thread-11 : {u'58442': u'58442', u'59509': u'59509', u'59354': u'59354'}
28-Dec-2017 15:53:29 - DEBUG   :: CP Server Thread-11 : Recieved OPDS command: AllTitles
28-Dec-2017 15:53:30 - DEBUG   :: CP Server Thread-11 : Recieved OPDS command: Comic
28-Dec-2017 15:53:32 - DEBUG   :: CP Server Thread-11 : Recieved OPDS command: AllTitles
28-Dec-2017 15:53:35 - DEBUG   :: CP Server Thread-11 : Recieved OPDS command: root
28-Dec-2017 15:53:35 - DEBUG   :: CP Server Thread-11 : {u'58442': u'58442', u'59509': u'59509', u'59354': u'59354'}
28-Dec-2017 15:53:36 - DEBUG   :: CP Server Thread-11 : Recieved OPDS command: Recent
```

**Comment by barbequesauce** [comment]
raw html, via curl:

```
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html>
<head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8"></meta>
    <title>500 Internal Server Error</title>
    <style type="text/css">
    #powered_by {
        margin-top: 20px;
        border-top: 2px solid black;
        font-style: italic;
    }

    #traceback {
        color: red;
    }
    </style>
</head>
    <body>
        <h2>500 Internal Server Error</h2>
        <p>The server encountered an unexpected condition which prevented it from fulfilling the request.</p>
        <pre id="traceback">Traceback (most recent call last):
  File "/opt/mylar/lib/cherrypy/_cprequest.py", line 645, in respond
    response.body = self.handler()
  File "/opt/mylar/lib/cherrypy/lib/encoding.py", line 188, in __call__
    self.body = self.oldhandler(*args, **kwargs)
  File "/opt/mylar/lib/cherrypy/_cpdispatch.py", line 29, in __call__
    return self.callable(*self.args, **self.kwargs)
  File "/opt/mylar/mylar/webserve.py", line 4871, in opds
    data = op.fetchData()
  File "/opt/mylar/mylar/opds.py", line 81, in fetchData
    result = methodToCall(**self.kwargs)
  File "/opt/mylar/mylar/opds.py", line 431, in _Recent
    if 'DateAdded' in issuebook.keys():
AttributeError: 'NoneType' object has no attribute 'keys'
</pre>
    <div id="powered_by">
    <span>Powered by <a href="http://www.cherrypy.org">CherryPy 3.2.0</a></span>
    </div>
    </body>
</html>
```

**Comment by barbequesauce** [comment]
Fixed with PR #1825
